src/helpers/GetDataType.js

```diff
@@ -21,7 +21,7 @@ function loadDependencies(def) {
 function getExpressionRefs(def) {
   return _.compact(
     mapDeep(def, (value, key) => {
-      if (value.libraryName && value.type === 'ExpressionRef' && (key === 'expression' || key === 'operand')) {
```

Comment:
hmm so I think getting rid of these booleans might hinder us looking for library references and lead to unexpected behavior. 

BUT. I think we can have the best of both worlds if we implement @okeefm's solution for passing in a function that returns a boolean to a unified mapDeep function that he suggested in the code review?? So maybe we can implement that suggestion alone in this PR instead. Thoughts?